mail_tracking/static/src/js/mail_tracking.js

```diff
@@ -0,0 +1,60 @@
+/* © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). */
+
+if (typeof window.jQuery === 'undefined') { throw new Error('Requires jQuery'); }
+(function ($, window, document) {
+    'use strict';
+
+    openerp.mail_tracking = function (instance) {
+        var _t = instance.web._t,
+            _lt = instance.web._lt;
+        var QWeb = instance.web.qweb;
+        var mail_orig = instance.mail;
+        var mail_inherit = function() {
+            instance.mail.MessageCommon.include({
+                init: function (parent, datasets, options) {
+                    this._super(parent, datasets, options);
+                    this.partner_trackings = datasets.partner_trackings || [];
+                }
+            });
+            instance.mail.ThreadMessage.include({
+                bind_events: function () {
+                    this._super();
+                    this.$('.oe_mail_action_tracking').on('click', this.on_tracking_status_clicked);
```

Comment:
But not at executing time. http://api.jquery.com/on/#event-handler says:

> When jQuery calls a handler, the this keyword is a reference to the element where the event is being delivered; for directly bound events this is the element where the event was attached and for delegated events this is an element matching selector. (Note that this may not be equal to event.target if the event has bubbled from a descendant element.) To create a jQuery object from the element so that it can be used with jQuery methods, use $( this ).